### PR TITLE
Home Page Updates

### PR DESCRIPTION
**Pull request by vnesslange**
If I learned anything from this project, its UI is much **harder** than it looks. 

The hero image was not working how I imagined, because the text was not clear on it. I made it half image/half description. 

It is probably a bit wordy for a normal Home Page, but for a School Project, It is an extension of our poster. 

ezgif com-video-to-gif (7)

### CODE REVIEW

**Comment by angieyeh** [comment]
Nice job! UI work is really challenging especially without someone giving us mocks to work with. I always have to look at other websites for inspiration. 

For example take a look at BW's landing page. The page is pretty simple. At the top there's an image of the website and then below there are some fun graphics that highlight the key features. If we had more time I would get a screen shot our app and make that the hero image, remove the security section, make the background white and use black text or the darker green color for certain text because green on green is slightly hard to read.  Given the time constraints, the main thing to do is to remove the security section because I think it's a lot to have that on a landing page.

**Comment by vnesslange** [comment]
@angieyeh Okay, I went ahead and removed the security part for now.

I tried to simplify based on your feedback quickly before I went to bed (lighter background, less crowded, dark text) would these changes be better? I can push these commits  if you prefer this:

ezgif com-video-to-gif (8)

Or we can just go with what's on this branch at the moment let me know.  I can push the commits before or after work tomorrow if you like this better!

**Comment by derthadams** [comment]
Good job, I like the direction it's going - simpler is better. 

If we're thinking of this more like a commercial site's homepage, the user probably doesn't need to know about session auth, React, Python/Django, AWS or any of those details, so I think the paragraphs starting "It securely..." and "JavaScript/React..." could largely be dropped.

There could be a simple call to action at the top - I think the user already knows the service is web based, so it could be "Key Fortress is a secure password manager",  or "Trust Key Fortress to guard your passwords", or something. Then there could be a short sentence underneath with maybe one or two very broad security details, like "zero-knowledge end-to-end encryption".

The list of features is good but I think it should stick to user-facing ones (I would delete "RESTful API design")

**Comment by vnesslange** [comment]
Updated

ezgif com-video-to-gif (9)

File: src/components/KeyFeatures.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                password
+              </li>
+              <li> Random password generator for creating strong passwords</li>
+            </ul>
+          </Typography>
+
+          <Box
+            sx={{
+              display: "flex",
+              flexDirection: "column-reverse",
```
[the remark this answers is not in the document]
Comment (vnesslange) [reply src/components/KeyFeatures.jsx:46]: It starts the content at the bottom of the column,

**Review by angieyeh — APPROVED** [review]
